include/mrdox/Metadata/Info.hpp

```diff
@@ -36,7 +35,19 @@ struct MRDOX_VISIBLE
 
     /** Kind of declaration.
     */
-    InfoKind Kind = InfoKind::Default;
+    InfoKind Kind;
+
+    /** Declaration access.
+
+        Class members use:
+        @li `AccessKind::Public`,
+        @li `AccessKind::Protected`, and
+        @li `AccessKind::Private`.
+
+        Namespace members use `AccessKind::None`.
+        Friends use `AccessKind::Friend`.
```

Comment:
umm... I'm not a fan of this. Why not just None

Reply:
I have to update this